app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt

```diff
@@ -394,6 +396,11 @@ class HomeFragment : Fragment() {
         requireComponents.core.engine.profiler?.addMarker(
             MarkersFragmentLifecycleCallbacks.MARKER_NAME, profilerStartTime, "HomeFragment.onCreateView",
         )
+
+        if (FeatureFlags.showWallpapers) {
+            val wallpaper = requireContext().settings().getWallpaper()
+            updateWallpaper(wallpaper, requireContext())
+        }
```

Comment:
Can you add a new line before this to space out the if statement block

Reply:
👍🏽